src/test/scala/wdl4s/wdl/values/WdlValueSpec.scala

```diff
@@ -50,7 +50,7 @@ class WdlValueSpec extends FlatSpec with Matchers {
     (WdlFloat(Double.MinValue), "-1.7976931348623157E308", { d: Double => d.isNegInfinity }))
 
   forAll(wdlFloatSpecials) { (wdlValue, rawString, validateFloat) =>
-    it should s"convert a special ${wdlValue.typeName} to/from raw string '$rawString'" in {
+    it should s"convert a special ${wdlValue.typeName} to/from raw string '$rawString'" ignore {
```

Comment:
Great catch!  I meant to drop a comment here and completely forgot.

The problem here was the old behavior at `WdlFloatType.fromWdlString` above.  This method takes a string and used to invoke `toFloat` rather than `toDouble`.  This is curious considering that the internal data type in a `WdlFloat` is in fact a `double`, and the compiler was silently widening the `float` into a `double` after this conversion anyway (I had to make this change because the new flags disapprove of these silent widenings).

These tests were validating that any values that fell between the limits of a `float` and a `double` were truncated by the `toFloat`.  I have no idea why that truncation behavior wasn't fixed rather than writing tests to validate that was happening.  I'd probably want to replace these with tests that validated truncation outside the limits of doubles, or just delete them altogether.